spec/integration/multiple_trackers_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe Mongoid::History do
+  before :all do
+    Mongoid::History.tracker_class_name = nil
+
+    class First
```

Comment:
Nitpick, these are global classes despite what you may think. I would put them into a `module` like `module MultipleHistoryTrackersSpec` to make sure there's no conflict, and make sure to fix up anything in other specs the same way as well.

Reply:
Oh i see. Yeah I was having conflict when I was using Foo. I'll do what you're suggesting.

Reply:
Yeah it's probably better to not make the default configuration optional and not rely on load order.

Reply:
So I did what you've asked but the tracker classes can't be put in a module as there's no way to specify the module when passing a tracker_class to #track_history.  I still moved the trackables into their own modules though.

I also tried doing the same thing for the other specs but even that wasn't working because those specs are using a modifier and there's no way to specify a module in the modifier.

I think we should leave in a separate issue since I don't have time right now to work on these. Let me know if you want me to revert the changes to the spec that I've added.